spec/onchain.tex

```diff
@@ -498,22 +594,60 @@ \subsection{Contest Transaction}\label{sec:contest-tx}
 	\[
 		s' > s
 	\]
+\item $\mathsf{ContestType} = \red{(\mathsf{UnusedInc}, \xi, \eta_\alpha)} \cup (\mathsf{UnusedDec}, \xi) \red{\cup (\mathsf{UsedInc}, \xi)}  \cup (\mathsf{UsedDec}, \xi, \eta_\omega)$, where $\xi$ is a multi-signature of the contesting snapshot \red{and $\eta_\alpha$} and $\eta_\omega$ are the digests of the UTxO to \red{increment or} decrement respectively, \red{four} cases are distinguished:
```

Comment:
See all comments from close above and make sure they are fixed here to (if they apply)